src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCache.java

```diff
@@ -317,26 +319,26 @@ private boolean satisfiedByCache(
         {
             final String name = h.name().asString();
             final String value = h.value().asString();
-            switch (name)
+            if (CACHE_CONTROL.equals(name))
             {
-            case CACHE_CONTROL:
                 // TODO remove need for max-age=0 (Currently can't handle multiple outstanding cache updates)
                 return value.contains(CacheDirectives.NO_CACHE) ||
                        value.contains(MAX_AGE_0) ||
                        value.contains(NO_STORE);
-            default:
-                return false;
             }
+            return false;
```

Comment:
Wouldn't this simplify to the following?
```
        return CACHE_CONTROL.equals(name) &&
            (value.contains(CacheDirectives.NO_CACHE) ||
            value.contains(MAX_AGE_0) ||
            value.contains(NO_STORE));
```

Reply:
yes.